src/components/common/Separator.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {
+    View
+} from 'react-native';
+import EStyleSheet from 'react-native-extended-stylesheet'
+import PropTypes from 'prop-types';
+
+const Separator = ({style}) => {
+    return (
+        <View style={[styles.defaultStyle, style]} />
+    );
+}
+
+const styles = EStyleSheet.create({
+    defaultStyle: {
+        borderBottomColor: '#cbcccb',
```

Comment:
Should this be a constant, or use one of the predefined ones in theme.js?

Reply:
Yeah, it looks like I use this color elsewhere, I'll pull it out